modules/installation-mirror-repository.adoc

```diff
@@ -33,20 +33,36 @@ x509: certificate relies on legacy Common Name field, use SANs or temporarily en
 
 Complete the following steps on the mirror host:
 
+[NOTE]
+====
+Optionally, you can use image tag name instead of image digest in the following commands.
+====
+
 . Review the
 link:https://access.redhat.com/downloads/content/290/[{product-title} downloads page]
-to determine the version of {product-title} that you want to install and determine the corresponding tag on the link:https://quay.io/repository/openshift-release-dev/ocp-release?tab=tags[Repository Tags] page.
+to determine the version of {product-title} that you want to install.
+The image release advisory includes the pullspec repository and digest. For example, link:https://access.redhat.com/downloads/content/290/ver={product-version}/rhel---8/[In the errata tab here],
+uses `quay.io/openshift-release-dev/ocp-release` for the pullspec repository.
 
 . Set the required environment variables:
-.. Export the release version:
+.. Export the image repository from the image release advisory:
```

Comment:
nit: Preexisting wording, so feel free to ignore, but `OCP_RELEASE_REPOSITORY=...` is setting a variable, not exporting it (which would be `export OCP_RELEASE_REPOSITORY=...`).  I'd use "Declare the image repository..." or something else to avoid saying "export" and risking confusion with `export`.  Also in this space [here][1] and discussion in #22008.

[1]: https://github.com/openshift/openshift-docs/pull/24611/files#diff-817deac03bba77d76c41d33355e9d880b59c57e2c88b87d21267256fc99818ebL35